app/mailers/user_mailer.rb

```diff
@@ -1,9 +1,38 @@
 class UserMailer < Devise::Mailer
   default from: "Chicago Tool Library <team@chicagotoollibrary.org>"
+  layout "mailer"
+
   after_action :store_notification
 
+  def initialize_from_record(record)
+    super # load user
+    @library = @user.library
+    @logo = :small
+  end
```

Comment:
Overriding this method feels a little gross, but due to the way these mailers are written I couldn't find a better way to do this.

Reply:
I starting writing something about code or tests that try to verify that the upstream interface hasn't changed, but then I found [it hasn't changed in 14 years](https://github.com/heartcombo/devise/blame/fec67f98f26fcd9a79072e4581b1bd40d0c7fa1d/lib/devise/mailers/helpers.rb#L22) so I think it's probably fine to just let our mailer previews and the existing unit tests cover this. 🙂